arrows/mvg/applets/bundle_adjust_tool.cxx

```diff
@@ -1277,6 +1277,11 @@ ::bundle_adjust_tool()
   : d( new priv( this ) )
 {}
 
+// ----------------------------------------------------------------------------
+bundle_adjust_tool
+::~bundle_adjust_tool()
+{}
```

Comment:
LGTM. In the past I've also been able to just write in the CXX the likes of `bundle_adjust_tool::~bundle_adjust_tool() = default;`, but there is certainly nothing wrong with this.

Reply:
Yeah, this may technically be a compiler bug (but I don't know the C++ standard well enough to be sure).